src/ast/ddl.rs

```diff
@@ -1098,17 +1098,116 @@ impl fmt::Display for ColumnOptionDef {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
```

Comment:
For the introduced options, can we add a short description on what they are and a link to the snowflake docs that mention where they come from? It'll make it easier to navigate later on when folks land on the struct/enum without prior context

Reply:
I added a reasonable documentation for structs with links to a Snowflake doc.